packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -87,6 +88,20 @@ export interface IEc2Service extends IService {
 
 }
 
+/**
+ * The properties to import from the service using the EC2 launch type.
+ */
+export interface Ec2ServiceAttributes {
+  /**
+   * The name of the cluster that hosts the service.
+   */
+  readonly clusterName: string;
+  /**
+   * The service ARN.
+   */
+  readonly serviceArn: string;
```

Comment:
There should also be `serviceName`, and one of `serviceArn` or `serviceName` is required.

Reply:
I added validation to prevent from set both and neither. If both are set, it may be confusing.